### ISSUE TITLE
[Accessibility- information structuring C9.3] Notes- Use of lists

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3050]
**Environment:** Tribe

**Step of reproduction:**

Access to the Home page of Notes or/and to the Notes tree

**Current behavior:**
All elements (Pages)  are not structured by lists

**Expected behavior:**

All pages should be structured into lists to allow quick consultation of content => Use ul/li lists to structure sequences of elements of the same nature, as well as for successive links

![Image](https://github.com/user-attachments/assets/d3beec09-3d6c-4364-96a3-efa632950cd2)
